Packs/SentinelOne/ReleaseNotes/3_2_0.md

```diff
@@ -0,0 +1,15 @@
+#### Integrations
+
+##### SentinelOne v2
+- Updated the *created_until* argument to be optional instead of required in the ***sentinelone-get-alerts*** command.
```

Comment:
Please update the REAME for these changes, i.e., for the args changed from "required" to "optional" and for others with value changes regarding the support for time format as "10 days" etc., so that its not contradicting

Reply:
Thanks for the feedback. Readme got updated with the latest commit.